repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthHandler.scala

```diff
@@ -1,11 +1,13 @@
 package org.openrepose.filters.rackspaceidentitybasicauth
```

Comment:
Removed all of the unnecessary empty parens on no parameter method calls and curly braces on case statements.